### DISCUSSION TITLE
Hey, I was trying to use this plugin with obsidian but  I am getting these errors when doing pip install

### DISCUSSION THREAD

**Discussion by Uchiha1391** [khoj-ai/khoj#179] [Q&A]
backround - I have never used python so it might be that I am doing something wrong. so give me some leeway for it.
here is the command I used and and the result.
![image](https://user-images.githubusercontent.com/28823508/219878292-b95bba77-38f8-4e40-9208-8e38f6a4c8f5.png)

![image](https://user-images.githubusercontent.com/28823508/219878263-ea1a7f77-14e6-4442-889e-b699420569f9.png)

**Comment by debanjum** [comment]
> I have never used python so it might be that I am doing something wrong. so give me some leeway for it

Hey @Uchiha1391, don't worry about it! And thanks for sharing screenshots of the issue (and your python version)

Khoj doesn't work with python 3.11 (yet). So to install the khoj server/backend, 
1. Install python version `3.10`, `3.9` or `3.8`. See [the official python for windows](https://www.python.org/downloads/windows/) page to install an appropriate python version (and associated pip).  E.g [latest python 3.10](https://www.python.org/downloads/release/python-31010/). See [sample python installation video](https://www.youtube.com/watch?v=Kn1HF3oD19c)
2. Then install khoj using the supported version of `python` (and `pip`) with `pip install khoj-assistant`

**Comment by debanjum** [comment]
Converting this into a discussion as this seems to be an issue with using an unsupported version of python (version 3.11)  to install Khoj. We can always open an issue if any concrete bugs are found with Khoj from this discussion
